kolibri/core/notifications/test/test_api.py

```diff
@@ -74,7 +75,6 @@ def setUp(self):
         self.contentnode_ids = [
             "25f32edcec565396a1840c5413c92451",
             "25f32edcec565396a1840c5413c92452",
-            "25f32edcec565396a1840c5413c92453",
```

Comment:
I assume this was just for simplicity of not having to make so many summary logs for completion?

Reply:
Hmm it was because this value wasnt being used anywhere (I thought I was reverted the deletion tho :sweat_smile:)

Reply:
Ah, if it was never used, no worries about clearing it up, just wanted to check why!